bin/create_encrypted_bucket.js

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
```

Comment:
It's a bash script calling itself after found the node or nodejs binary :) pretty cool :)